src/ucp/wireup/wireup_ep.h

```diff
@@ -39,7 +39,7 @@ struct ucp_wireup_ep {
     volatile uint32_t         pending_count; /**< Number of pending wireup operations */
     volatile uint32_t         flags;         /**< Connection state flags */
     uct_worker_cb_id_t        progress_id;   /**< ID of progress function */
-    ucp_ep_params_t           ucp_ep_params; /**< UCP EP paramters */
+    unsigned                  ep_init_flags; /**< UCP wireup EP init flags */
```

Comment:
i don't think we need to change all flows to pass err handling mode by ep_init_flags
instead keep err handling mode in this struct (instead of params), and in ucp_cm_ep_client_do_initial_config() create params struct based on this field here

Reply:
why? let's do it unified. actually ucp_ep_params is not needed in any flow.
ep_init_flags are common and extendable enough for now.

Reply:
but error handling mode parameter is theoretically an enum which can have more than 2 values
by making it bit field you will only support 2 values
it should be equivalent to ucp_wireup_sockaddr_data_t::err_mode

Reply:
we can use more that 1 bit, if we don't have space here, we always can separate it because these flags are internal and local.